test/test_myers.py

```diff
@@ -0,0 +1,153 @@
+import unittest
+import os
+from random import randint
+
+from longest_common_subsequence.myers import (
+    longest_common_substring, brute_force_lcs)
+from generator.rand import random_word
+
+def validate_text_lcs(text, text_len, lcs):
+  i = 0
+  lcs_len = len(lcs)
+  for j in range(1, text_len + 1):
+    if i != lcs_len and text[j] == lcs[i]:
+      i += 1
```

Comment:
Alternatively you may use:
def validate_text_lcs(x, y):
    it = iter(y)
    return all(any(c == ch for c in it) for ch in x)
Since we use the same "it" over the whole second line, it should use all letters of y only once and in proper order.